### PR TITLE
click-aliases pypi package

### PR DESCRIPTION
**Pull request by tedil**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by tedil** [comment]
@conda-forge/help-python Ready for review

File: recipes/click-aliases/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    - click_aliases
+
+about:
+  home: "https://github.com/click-contrib/click-aliases"
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: "Enable aliases for Click"
+  doc_url: 
+  dev_url:
```
Comment (scopatz) [inline recipes/click-aliases/meta.yaml]: Please remove empty entries

File: recipes/click-aliases/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: f48012077e0788eb02f4f8ee458fef3601873fec6c998e9ea8b4554394e705a3
+
+build:
+  number: 0
```
Comment (scopatz) [inline recipes/click-aliases/meta.yaml]: Can this be `noarch: python`?

Comment (tedil) [reply recipes/click-aliases/meta.yaml]: Yes, I think so.

**Pull request merged**